codingtovictory/day6/dailyChallenge/index.html

```diff
@@ -0,0 +1,23 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Day6DailyChallenge</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h2>Email Link</h2>
+    <a href="mailto: tialee_93@hotmail.com">Send Email</a>
```

Comment:
@Xensen93 space is not needed here but will still work

Reply:
Okay thank you!